src/common/tools.ts

```diff
@@ -369,3 +369,8 @@ export const getLocalMessage = (path: string) => {
 
 export const URLReg =
   /https?:\/\/(www\.)?[-a-zA-Z0-9@:%._\\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_\\+.~#?&//=]*)/
+
+export const getDocLinkByVersion = (version: string) => {
+  const lang = i18n.global.locale === 'zh' ? 'zh' : 'en'
+  return `https://www.emqx.io/docs/${lang}/v${version}/`
```

Comment:
Not the document link, view the changelog page. View the changelog page.

https://www.emqx.com/zh/changelogs/broker/5.0.6


`https://www.emqx.com/${lang}/changelogs/broker/${version}`